src/jit/codegencommon.cpp

```diff
@@ -7619,7 +7619,22 @@ void CodeGen::genPrologPadForReJit()
 
 #ifdef _TARGET_XARCH_
     if (!(compiler->opts.eeFlags & CORJIT_FLG_PROF_REJIT_NOPS))
+    {
         return;
+    }
+
+#if FEATURE_EH_FUNCLETS
+
+    // No need to generate pad (nops) for funclets.
+    // When compiling the main function (and not a funclet)
+    // the funCurrentFunc->funKind == FUNC_ROOT.
```

Comment:
Minor nit - "the" in the last line of comment is not needed.

_// When compiling the main function (and not a funclet), 
// funCurrentFunc->funKind == FUNC_ROOT_